src/services/TwitchService.ts

```diff
@@ -62,15 +65,57 @@ export class TwitchService implements ITwitchService {
     this._client.say(this._configuration.Twitch.Channel, message);
   }
 
-  public AddCommand(command: ICommand) : ITwitchService {
+  public AddCommand(command: ICommand): ITwitchService {
     this._commands.push(command);
     return this;
   }
 
-  public AddScheduler(scheduler: IScheduler) : ITwitchService {
+  public AddScheduler(scheduler: IScheduler): ITwitchService {
     setInterval((s) => {
       s.Action(this);
     }, scheduler.Minutes * 60000, scheduler);
     return this;
   }
+
+  private _getEventParams(eventType: EventType, ...args: any[]): any {
```

Comment:
Move this methods on mappers or helpers. 
You can also create `TwitchClient` class to wrappe tmi.js